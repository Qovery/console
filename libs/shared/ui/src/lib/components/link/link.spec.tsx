import { render } from '@testing-library/react'

import Link from './link'

describe('Link', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Link />)
    expect(baseElement).toBeTruthy()
  })
})
