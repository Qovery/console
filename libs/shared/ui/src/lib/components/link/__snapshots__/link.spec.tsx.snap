// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalLink should match with ExternalLink component 1`] = `
<div>
  <a
    class="cursor-pointer transition duration-100 font-medium inline-flex items-center gap-1 text-sky-500 hover:text-sky-600 text-sm"
    href="https://qovery.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    My external link
    <i
      aria-hidden="true"
      class="fa-solid fa-arrow-up-right-from-square text-xs leading-5"
    />
  </a>
</div>
`;

exports[`Link should match with Link component 1`] = `
<div>
  <a
    class="cursor-pointer transition duration-100 font-medium inline-flex items-center gap-1 text-sky-500 hover:text-sky-600 text-sm"
    href="/"
  >
    My link
  </a>
</div>
`;
